src/main/kotlin/uk/gov/justice/digital/hmpps/nomisprisonerapi/jpa/AgencyLocation.kt

```diff
@@ -49,75 +49,11 @@ data class AgencyLocation(
   @Convert(converter = YesNoConverter::class)
   val active: Boolean = false,
 
-  @Column(name = "LONG_DESCRIPTION")
-  val longDescription: String? = null,
-
-  //    @OneToMany(mappedBy = "agencyLocId", cascade = CascadeType.ALL)
-  //    @Default
-  //    private List<AgencyLocationEstablishment> establishmentTypes = new ArrayList<>();
-  //
-  //    @OneToMany(mappedBy = "agencyLocation", cascade = CascadeType.ALL, fetch = FetchType.LAZY, orphanRemoval = true)
-  //    @Default
-  //    private List<CaseloadAgencyLocation> caseloadAgencyLocations = new ArrayList<>();
   @Column(name = "DEACTIVATION_DATE")
   val deactivationDate: LocalDate? = null,
 
-  @ManyToOne(fetch = FetchType.LAZY)
```

Comment:
removed all the commented out code which is confusing and the fields never used any where